src/_ert_job_runner/client.py

```diff
@@ -73,7 +75,7 @@ def __init__(
         self._max_retries = max_retries
         self._timeout_multiplier = timeout_multiplier
         self.websocket: Optional[WebSocketClientProtocol] = None
-        self.loop = asyncio.new_event_loop()
+        self.loop = new_event_loop()
```

Comment:
Unfortunately, we'd like to avoid using ert in job_runner.

Reply:
Even the code in `_ert`?

Reply:
As discussed, that is fine in the end. So just ignore my comment above :)